autoscaler/controllers/datacollection/configmap.go

```diff
@@ -139,6 +145,24 @@ func getConfigMapData(apps *odigosv1.InstrumentedApplicationList, dests *odigosv
 	}
 	processorsCfg["resourcedetection"] = config.GenericMap{"detectors": []string{"ec2", "gcp", "azure"}}
 
+	exporters := config.GenericMap{
+		"otlp/gateway": config.GenericMap{
+			"endpoint": fmt.Sprintf("dns:///odigos-gateway.%s:4317", env.GetCurrentNamespace()),
+			"tls": config.GenericMap{
+				"insecure": true,
+			},
+		},
+	}
+	tracesPipelineExporter := []string{"otlp/gateway"}
```

Comment:
do we use this exporter if `setTracesLoadBalancer` is true?

Reply:
Yes in case metrics/logs also configured